src/pymovements/gaze/io.py

```diff
@@ -78,7 +78,17 @@ def from_csv(
         the column will be used for pixel to dva transformations. If not specified, the
         constant eye-to-screen distance will be taken from the experiment definition.
     **read_csv_kwargs:
-            Additional keyword arguments to be passed to polars to read in the csv.
+        Additional keyword arguments to be passed to :py:func:`polars.read_csv` to read in the csv.
+
+        For example, if you want to read a csv file with a custom separator, you can pass
```

Comment:
Can you put these three examples into the example section?
You can use the judo example file under `tests/files` for the first one, and maybe a different one for the last example?